integration-tests/tests/run-rstudio.js

```diff
@@ -0,0 +1,66 @@
+// This test is owned by the Interactive Analysis (IA) Team.
+const _ = require('lodash/fp')
+const { withRegisteredUser, withBilling, withWorkspace, performAnalysisTabSetup } = require('../utils/integration-helpers')
+const {
+  click, clickable, findElement, noSpinnersAfter, fillIn, findIframe, findText, dismissNotifications, getAnimatedDrawer, image, input
+} = require('../utils/integration-utils')
+const { registerTest } = require('../utils/jest-utils')
+
+
+const rFileName = 'test-rmd'
+
+const testRunRStudioFn = _.flow(
+  withWorkspace,
+  withBilling,
+  withRegisteredUser
+)(async ({ workspaceName, page, testUrl, token }) => {
+  await performAnalysisTabSetup(page, token, testUrl, workspaceName)
+
+  // Create analysis file
+  await click(page, clickable({ textContains: 'Start' }))
+  await findElement(page, getAnimatedDrawer('Select an application'))
+  await click(page, image({ text: 'Create new R markdown file' }))
+  await fillIn(page, input({ placeholder: 'Enter a name' }), rFileName)
```

Comment:
Do we not have to select a `File Type`? 

<img width="433" alt="Screen Shot 2022-06-29 at 4 55 50 PM" src="https://user-images.githubusercontent.com/23626109/176543247-1c8584cd-2048-46b2-8fe8-2949eb4abf54.png">

Reply:
No, I just use the default value here. This isn't valid for file name since there is no default